### DISCUSSION TITLE
There is a large gap between the size of the result converted by babel and the result converted by babel-loader in webpack

### DISCUSSION THREAD

**Discussion by 1541157892** [babel/babel#15652] [Q&A] [answered]
Hello, I'm now running into something that's beyond my comprehension.
The same code I directly use babel to transform the output result is only a few dozen lines, but I use babel-loader in webpack to convert the output result is as high as thousands of lines. I initially thought it might be because babel did not process the require files during the conversion process, and webpack needs to extract all the files in require. I don't know if my guess is correct. If I can get the correct answer, I will Will be very grateful.

**Comment by nicolo-ribaudo** [comment, marked as answer]
It's because webpack also includes all your dependencies in the same file, while Babel doesn't.

**Reply by 1541157892** [reply]
Thank you, after reading your answer, I can confirm that my guess should be correct
